### PR TITLE
nvuApp: css improvement

### PR DESCRIPTION
**Pull request by VasilMitovClouway**
Improvement on the CSS part of the app from me and Ianislav.

### CODE REVIEW

**Comment by p-nedelchev** [comment]
Please add some screenshots.

**Comment by VasilMitovClouway** [comment]
Due to github problems we were unable to add screenshots to the PR.

**Comment by qnislavnachev** [comment]
There are some pics of our work today

create question

createtest

login

questionlist

screenshot from 2017-02-01 15-27-43

testslist

File: src/main/resources/com/clouway/nvuapp/adapter/http/servlet/userQuestionFilter.ftl [hunk 1]
```diff
@@ -1,6 +1,7 @@
+<div class="selectFilter">
 <p style="display:inline">Категория</p>
-<select name="category">
-    <option value="non">Неизбрано</option>
+<select class="select" name="category">
```
Comment (mgenov) [inline src/main/resources/com/clouway/nvuapp/adapter/http/servlet/userQuestionFilter.ftl]:
```html
<select class="select"> 
``` 
doesn't sound good. 

```html
<select class="dropDownMenu">
... 
<select class="dropDownBox">
...
<select class="selectionBox">
```

**Comment by VasilMitovClouway** [comment]
fixed name of class in css file

**Pull request merged**